assets/text/effects/genericCampaignPlots_chapter1_anyEnemy_omen_clearAll_boldAction.properties

```diff
@@ -0,0 +1,25 @@
+~01~prompt~panel_007.(foes_Morthagi)~4_narration=<foes.cdgmt:The cultist roves the rubble and coals, casting about with a long stone blade./The Drauv throws a fire-catching gaze across the devastation, plucking a hungry bowstring./The disfigured beast bellows, and gurgles, and canters about, leaving destruction wherever its hooves fall./The bone-creature scuttles and rips down burning beams, intent on destruction, dragging fire wherever it creeps./With dragonly howls, it harries its fleeing victims, spreading death and flames, here to dismantle all human creation.>
+~01~prompt~panel_008.(foes_Thrixl)~4_narration=<foes.cdgmt:The cultist roves the rubble and coals, casting about with a long stone blade./The Drauv throws a fire-catching gaze across the devastation, plucking a hungry bowstring./The disfigured beast bellows, and gurgles, and canters about, leaving destruction wherever its hooves fall./The bone-creature scuttles and rips down burning beams, intent on destruction, dragging fire wherever it creeps./With dragonly howls, it harries its fleeing victims, spreading death and flames, here to dismantle all human creation.>
+~01~prompt~panel_009~7_mystic=<mystic.personality:Grab your weapon, <warrior>. The library's not on fire, but we need speed, haste, uh... we need courage? We can be [i]dauntless,[] right?/I'd say we should run, but I don't think we've got a choice.[face:interested] Time to grab your weapon, I think./Grab your weapon, <warrior>! [face:interested]Or are you just going to stand there [i]quacking[] like the first duck of spring?/Think about this. [i]Everything[] is about to burn. If you want to save your own stuff, then grab your weapon.[face:grim]/[face:grim]This is going to be a hard day, so stay strong. I'll be here, <hunter>'ll be here. Grab your weapon, <warrior>./Go on and grab your weapon, <warrior>! This's the day we show our mettle! Don't you [i]feel it?[] This's the day we're really [i]born![][face:joke]/Grab your weapon, <warrior>. [face:grim]We're going into a [i]battle,[] now. One like the book-heroes have. And I won't let us lose it./Grab your weapon, <warrior>. Or stay out of the way.[face:grim]/Sometimes we only learn to swim when we fall in the water. And sometimes, that water's [i]boiling.[] Grab your weapon, <warrior>.[face:talking]/[face:talking]I know you won't maybe understand this, but we're getting our [i]calling,[] now. Grab your weapon, <warrior>./Grab your weapon, <warrior>. [face:dubious]Unless you want to try [i]convincing[] these flames to quit?>
+~01~prompt~panel_010.(legend_is_missing)~1_warrior=My... [i]weapon[]...?
+~02~choice_one~panel_001~1_warrior=[face:talking]Well. Having reach is good. \n<foes.cdgmt:I'm not here to hug cultists./Don't want to hug <warrior.drauven:the enemy/<hunter.drauven:the enemy/<mystic.drauven:the enemy/a Drauv>>>./I'm no monster-hugger./I'm no monster-hugger./I'm no monster-hugger.>
```

Comment:
> `<warrior.drauven:the enemy/<hunter.drauven:the enemy/<mystic.drauven:the enemy/a Drauv>>>`

🤔 A more compact way to write this might be something like:
```<test:warrior.drauven+hunter.drauven+mystic.drauven:the enemy/a Drauv>```

Alternatively, we could add a role to the comic that selects a Drauven character (since I think the 5-character "multiplayer" scenarios also use these events?) - maybe as the "friend" role or whatever isn't being used already in the comic. Then we could just use:

```<friend.exists:the enemy/a Drauv>```

Reply:
Ah, I was thinking of doing a test variable like that but didn't realise it would work with the <> brackets, so thanks for letting me know!